Focusgroup/explainer.md

```diff
@@ -61,73 +60,93 @@ implementing the pattern:
-       accessibility patterns.
+    * Configuration is managed in one place.
+    * Provide easy to understand usage into HTML and CSS patterns.
+    * Avoid "spidery connections" e.g., using IDRefs or custom names that are hard to maintain.
+3. Complimentary declarative representations in HTML and CSS
+    * HTML attributes offers focusgroup usage directly with impacted content and provide for
+       the most straightforward scenarios.
+    * CSS properties allow for responsive design patterns to conditionally apply focusgroup 
+       behavior under changing layouts. Enables some advanced use cases where selector-based
+       matching is advantageous.
      
 ## 5. Use Cases
 
-1. A set of focusable controls parented by one element can be trivially added to a focusgroup.
-2. A set of focusable controls spanning a hierarchy of DOM can be added to a single logical 
-    focusgroup.
-3. A logical focusgroup can be configured to have wrap-around focus semantics if desired.
-4. A focusgroup can be configured to respond to either horizontal navigation keys or vertical
-    keys or both (to trivially reserve one axis of arrow key behavior for supplementary
-    actions, such as opening nodes in a tree view control).
-5. Focusgroups can be nested to provide logical navigation into multiple composed widgets (such
-    as lists within lists).
-6. Focusgroups can be used for grid-type navigation (structured content grids like &lt;table&gt;,
-    not CSS grids).
-
-## 6. Proposed Solution
-
-A new HTML attribute 'focusgroup'.
-
-The presence of this attribute on an element<sup><a href="#note1">1</a></sup> allows any of its
-focusable children (including those with `tabindex=-1`) to be focused using the keyboard arrow keys.
+1. (Child opt-in) Group a set of focusable child elements under a single focusgroup.
+2. (Descendent opt-in) Focusable elements deeply nested can participate in a single focusgroup.
+3. (Wrap) Focusgroup can be configured to have wrap-around focus semantics.
+4. (Horizontal/vertical) A focusgroup can be configured to respond to either horizontal navigation
+    keys or vertical keys or both (to trivially reserve one axis of arrow key behavior for 
+    supplementary actions, such as opening nodes in a tree view control).
+5. (Extend same direction) Focusgroups can be nested to provide arrow navigation into multiple 
+    composed widgets (such as lists within lists).
+6. (Extend opposite direction) Focusgroups can be nested to provide arrow navigation into composed
+    widgets with orthogonal navigation semantics (such as horizontal-inside-vertical menus).
+7. (Multiple focusgroups) Multiple focusgroups can be established on a single element (advanced CSS 
+    scenario).
+8. (Opt-out) Individual elements can opt-out of focusgroup participation (advanced CSS scenario)
+9. (Grid) Focusgroups can be used for grid-type navigation (structured content grids like &lt;table&gt;,
```

Comment:
"like <table>" implies that other table-like structures might accept grid focusgroups, but per our latest discussion, only <table> should support grid focusgroups.